sdk/rust/src/processor/handler.rs

```diff
@@ -389,7 +389,7 @@ impl TransactionContext {
         let x: &[u8] = &serialized;
 
         let mut future = self.sender.send(
-            Message_MessageType::TP_RECEIPT_ADD_DATA_REQUEST,
```

Comment:
This fix should be broken out in its own commit

Reply:
Addressed.